aadl.library/Deployment_Properties.sysml

```diff
@@ -0,0 +1,17 @@
+/* SysML v2 Domain Library for AADL:
+ * references: 
+ * SAE AS-5506D AADL appendix A1
+ * OMG SysML v2 release 2024-02 */
+ 
+ standard library package Deployment_Properties {
+	
+	import AADL_Project::*;
+	
+	allocation def Actual_Processor_Binding {

```

Comment:
Please add a comment stating that this is a possible approach for handling AADL binding properties and that we need to have further discussions about it.

Reply:
No problem to add comments here and in other places too.